diff --git a/node_modules/react-native-zoom-us/android/build.gradle b/node_modules/react-native-zoom-us/android/build.gradle
index 1c8cbb5..9c6c86e 100644
--- a/node_modules/react-native-zoom-us/android/build.gradle
+++ b/node_modules/react-native-zoom-us/android/build.gradle
@@ -3,7 +3,7 @@ apply from: 'zoom-meeting-sdk-versions.gradle'
 
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
         maven {
             url 'https://maven.google.com/'
             name 'Google'
